internal/manifest/v1alpha/examples/labels.go

```diff
@@ -16,3 +16,12 @@ func exampleLabels() v1alpha.Labels {
 		"area":   {"latency", "slow-check"},
 	}
 }
+
+func exampleCompositeLabels() v1alpha.Labels {
+	return v1alpha.Labels{
+		"team":   {"green", "ux"},
+		"env":    {"prod", "dev"},
+		"region": {"us", "eu"},
+		"area":   {"user-experience"},
+	}
+}
```

Comment:
since this is only used in one place by composite SLO, it doesn't need a separate function, you can just inline it in the generation code, or move the function to `slo_composite.go` file